nhrpd/nhrp_vty.c

```diff
@@ -459,6 +460,54 @@ DEFUN(if_no_nhrp_holdtime, if_no_nhrp_holdtime_cmd,
 	return CMD_SUCCESS;
 }
 
+#define NHRP_CISCO_PASS_LEN 8
+DEFUN(if_nhrp_authentication, if_nhrp_authentication_cmd,
+      "nhrp authentication PASS",
+      NHRP_STR
+      "Specify plaint text password used for authenticantion\n"
+      "Password, plain text, limited to 8 characters\n")
+{
+	VTY_DECLVAR_CONTEXT(interface, ifp);
+	struct nhrp_cisco_authentication_extension *auth;
+	struct nhrp_interface *nifp = ifp->info;
+	const char *pass = argv[2]->arg;
+	int pass_len = strlen(pass);
+
+	if (pass_len > NHRP_CISCO_PASS_LEN) {
+		vty_out(vty, "Password size limit exceeded (%d>%d)", pass_len,
```

Comment:
new line is missing for vty_out()'s output.

Reply:
fixed